webapp/templates/partials/scoreboard.html.twig

```diff
@@ -35,7 +35,12 @@
                     {% set now = 'now'|date('U') %}
                     {{ now | printremainingminutes(current_contest.endtime) }}
                 {% else %}
-                    starts: {{ current_contest.starttime | printtime }} - ends: {{ current_contest.endtime | printtime }}
+                    {% if current_contest.freezeData.started %}
+                        started:
+                    {% else %}
+                        starts:
+                    {% endif %}
+                    {{ current_contest.starttime | printtime }} - ends: {{ current_contest.endtime | printtime }}
```

Comment:
Should the same not be done for "ends"?

Reply:
Once the contest has ended, we match any of the above cases (contest over, or final standings)